source/beamline/SGM/SGMHexapodTransformedAxis.cpp

```diff
@@ -185,6 +200,111 @@ AMAction3 * SGMHexapodTransformedAxis::createSetParameterActions(double startPoi
 	return action;
 }
 
+AMAction3 *SGMHexapodTransformedAxis::createSetParametersActions(const QVector3D &startPoint, const QVector3D &endPoint, double deltaTime)
```

Comment:
What are you hoping for with this action to set up moves in the native coordinate system?
I'm not sure when these would be used.